Wireframe/README.md

```diff
@@ -25,15 +25,15 @@ Use the wireframe sketch provided to lay out the page. Please note that this is
 
 ## Acceptance Criteria
 
-- [ ] The webpage has a consistent color scheme for links and text.
-- [ ] Semantic HTML tags are used to structure the webpage.
-- [ ] The page header includes a title and description.
-- [ ] The articles section has three articles, each including a title, summary, and a link.
-- [ ] The page footer is fixed to the bottom of the viewport.
-- [ ] The first article is styled using the :first-child pseudo-class.
-- [ ] The webpage is styled using a linked .css file.
-- [ ] The webpage is properly committed and pushed to a branch on GitHub.
-- [ ] The webpage does not use any frameworks such as Bootstrap.
+- [ ✅ ] The webpage has a consistent color scheme for links and text.
```

Comment:
BTW if you tick the boxes on the GitHub interface it will show as completed tasks.  Look at the view on https://github.com/CodeYourFuture/Module-User-Focused-Data/pull/194